pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -1770,9 +1794,9 @@ status:
 			}
 
 			gotTimeoutValue := postReconcileRun.GetTimeout()
-			expectedTimeoutValue := time.Second
+			expectedTimeoutValue := time.Hour
 
-			if d := cmp.Diff(gotTimeoutValue, expectedTimeoutValue); d != "" {
+			if d := cmp.Diff(expectedTimeoutValue, gotTimeoutValue); d != "" {
 				t.Fatalf("Expected timeout for created Run, but got a mismatch %s", diff.PrintWantGot(d))
```

Comment:
I think this block should be removed. We are no longer changing the timeout for the run after it's created.

Reply:
👍